src/modal/modal.spec.ts

```diff
@@ -7,7 +7,7 @@ import {NgbModalModule, NgbModal, NgbActiveModal, NgbModalRef} from './modal.mod
 const NOOP = () => {};
 
 @Injectable()
-class SpyService {
+export class SpyService {
```

Comment:
At some point, the Angular compiler was yelling at me for this.
But it was a bad configuration where tests were compiled, not needed anymore.